pkgs/development/python-modules/pytest-pudb/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+, pudb
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-pudb";
+  version = "0.7.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-DqhzFtOcghY9NAwooWjgihY7jT9ITmClPJ/V7v5DLGM=";
+  };
+
+  buildInputs = [ pytest ];
+
+  propagatedBuildInputs = [ pudb ];
+
+  pythonImportsCheck = [ "pytest_pudb" ];
+
```

```suggestion

  # upstream has no tests
  doCheck = false;
  
```
Please add `doCheck = false` with the reason.

---

This package has tests that are on the github repo but the author doesn't use tags for the release so I would have to hard code the rev to a6b3d2f. Is it ok to use the direct ref? If so I'll update to pull from there and add pytestCheckHook

---

Oh, I missed the test file.
I don't think it's a problem to hardcode the git hash like this since we do that sometimes.
This package already has an [issue](https://github.com/wronglink/pytest-pudb/issues/24), so it would be better to mention it.